opentelemetry/chart/alerts/collector-alerts.yaml

```
@@ -1,7 +1,7 @@
 groups:
   - name: collector-alerts
     rules:
-  {{- if not (.Values.prometheusRules.disabled.FilelogRefusedLogs | default false) }}
+  {{- if not (.Values.opentelemetry.prometheus.rules.disabled.FilelogRefusedLogs | default false) }}
```

This is pretty hard to read, suggesting to simplify this to

```suggestion
  {{- if not ( has 'FilelogRefusedLogs' .Values.opentelemetry.prometheus.rules.disabled }}
```
and use it as sort of a _blacklist_.

`Values.opentelemetry.prometheus.rules.disabled` needs to be used as a list, instead of dict then.

Applies to all alerts here.

---

Agreed, that is clearer to read.

To make the `.Values....rules` clear what is your take on adding this to the `Values.yaml`